### PR TITLE
Bug fix in mortality concepts

### PR DESCRIPTION
**Pull request by mcwimm**
Getter and setter functions were mixed up.

### CODE REVIEW

File: TreeModelLib/TreeModel.py [hunk 1]
```diff
@@ -24,34 +24,37 @@ def iniMortalityConcept(self, args):
...
-    ## This function calls getSurvival() of all selected mortality
+    ## This function calls setSurvive() of all selected mortality
     # concepts and checks if the conditions for death are met.
-    def getSurvival(self):
+    def setTreeKiller(self):
         survive = []
         for mortality_concept in self.mortality_concept:
-            survive.append(mortality_concept.getSurvival(self))
+            mortality_concept.setSurvive(self)
+            survive.append(mortality_concept.getSurvive())
```
Comment (jbathmann) [inline TreeModelLib/TreeModel.py:57]: kill 'em all!
The name sounds funny, but ok. :)

Comment (jbathmann) [reply TreeModelLib/TreeModel.py:57]: You changed getSurvial to get-survive. Does still still work with all the other concepts?
Sorry but I am a bit confused here.
You could try to start the tests in /test/*.py to check, whether it messes up anything. However, you probably already did this?

**Review by jbathmann — COMMENTED** [review]
Looks good

**Comment by jbathmann** [comment]
What do you think of making the variables private, which have are used by the getFunctions?

variable -> _variable

File: TreeModelLib/TreeModel.py (hunk 1 above)
Comment (mcwimm) [reply TreeModelLib/TreeModel.py:57]: I was confused too :)
In this case I changed the name to getSurvive as the function returns the variable self. survive...

The code is tested and compared with results from a previous simulation study... the tests in test/ are not helpful for this purpose atm. This should change after the implementation of ModulBenchmarks as tests.

**Comment by mcwimm** [comment]
> What do you think of making the variables private, which have are used by the getFunctions?
> 
> variable -> _variable

Good idea but would be helpful to talk about this before (today?)

File: TreeModelLib/TreeModel.py (hunk 1 above)
Comment (jbathmann) [reply TreeModelLib/TreeModel.py:57]: :+1:

**Comment by mcwimm** [comment]
> What do you think of making the variables private, which have are used by the getFunctions?
> 
> variable -> _variable

done

**Pull request merged**